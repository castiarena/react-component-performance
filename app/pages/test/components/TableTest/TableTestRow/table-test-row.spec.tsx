import React from 'react';
import TableTestRow from '.';
import renderWithTheme from '../../../../../testing-tools/renderWithTheme';
import {Passenger} from '../../../gateway/passenger.gateway';

describe('', () => {
    it('', () => {
        const mockPassenger: Passenger = {
            _id: 'somemockid',
            name: 'John Wick',
            airline: {
                _id: 0,
                name: 'Reeves airline',
                logo: 'some',
                slogan: 'I\'m thinking I\'m back',
                // eslint-disable-next-line camelcase
                head_quaters: 'New york',
                website: 'https://mr-reeves.com/',
                established: 'no idea',
            },
            trips: Infinity,
        };
        const mockData: { items: Passenger[]} = {
            items: [
                mockPassenger,
            ],
        };
        const { asFragment } = renderWithTheme(
            <TableTestRow data={mockData} index={0} style={{}} />,
        );
        expect(asFragment()).toMatchSnapshot();
    });
});
