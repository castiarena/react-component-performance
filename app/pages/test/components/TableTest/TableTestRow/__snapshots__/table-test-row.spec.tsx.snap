// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` 1`] = `
<DocumentFragment>
  <div
    class="sc-pNWxx iFjwNM"
  >
    <div
      class="sc-bdnylx sc-kEqYlL cDeQyM gbiHYP"
    >
      <div
        class="sc-gKAblj sc-fujyUd kYlZCs grzqIF"
      >
        <img
          alt="Reeves airline"
          class="sc-gtssRu sc-iCoHVE gOofrc cHbJeS"
          src="some"
        />
        <p
          class="sc-eCApGN sc-jrsJCI inMmBm kDVMpm"
        >
          <a
            class="sc-jSFkmK eWscpE"
            href="https://mr-reeves.com/"
            target="_blank"
          >
            Reeves airline
          </a>
        </p>
        <p
          class="sc-eCApGN sc-jrsJCI inMmBm kDVMpm"
        >
          Infinity
        </p>
        <p
          class="sc-eCApGN sc-jrsJCI inMmBm kDVMpm"
        >
          John Wick
        </p>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
